enplot/run.py

```diff
@@ -148,7 +148,7 @@ def main():
             if args.debug:
                 print("Processing data file " + data_file)
 
-            M, m, n = enplot.base.file_data_read(data_file)
+            M, m, n = enplot.base.file_data_read(data_file, usenp=True)
```

Comment:
I think this is good. Let's make np.getfromtxt the new default reading method, while keeping the old one as optional for now. eventually the old method can be dropped if there are no unexpected regressions.